src/jagadeeshMJun20/BankingApp.java

```diff
@@ -0,0 +1,46 @@
+package jagadeeshMJun20;
+
+public class BankingApp {
+	static int debitcount;
+	static int creditcount;
+	static int statementcount;
+	int indDebCount;
+	int indCredCount;
+	int indStatementCount;
+		
+		void  debitAmout(){
+		  debitcount++;
+		  indDebCount++; 
+		}
+		void creditAmout(){
+		creditcount++;
+		indCredCount++;
```

Comment:
You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.

Reply:
Updated the code as per the comments.